Modules/expat/xmlparse.c

```diff
@@ -85,6 +85,7 @@
 #ifdef _WIN32
 #  define getpid GetCurrentProcessId
 #else
+// HAVE_SYS_TIME_H is enforced by configure
```

Comment:
AFAIK, this will go away next time our copy of `expat` is updated.

Reply:
Good point, I'll revert this as well.